.expeditor/verify.pipeline.yml

```diff
@@ -221,22 +221,6 @@ steps:
         limit: 1
 
   - label: "[unit] :linux: sup"
```

Comment:
Did we intend to remove the inconsistent sup tests? I assume yes.

Reply:
Yes. It's possible we could add them back later if they prove to be unrelated to the locking issue, but hopefully this will change them from being inconsistent passing tests to consistently passing or failing, which we can deal with more directly.